op-e2e/Makefile

```diff
@@ -4,6 +4,12 @@ test:
 lint:
 	golangci-lint run -E goimports,sqlclosecheck,bodyclose,asciicheck,misspell,errorlint -e "errors.As" -e "errors.Is"
 
+test-external-%: external_%/shim
```

Comment:
My make skills aren't terrific, but, the idea here is that if anyone drops another `external_<name>` directory into the `op-e2e` folder, they can execute the test with:

```
make test-external-<name>
```